File: C_Programming/Assignment/Assignment3/range.c
#include <stdio.h>

void main() {
    int start, end, sum = 0, current;
    printf("Enter the start of the range: ");
    scanf("%d", &start);
    printf("Enter the end of the range: ");
    scanf("%d", &end);
    current = start;
    while (current <= end) {
        sum =sum+ current;  
        current++;      
    }
    printf("Sum of numbers from %d to %d is: %d\n", start, end, sum);

    
}

//output:Enter the start of the range: 2
//Enter the end of the range: 45
//Sum of numbers from 2 to 45 is: 1034

File: Java_Programs/Assignment3/Range.java
package Assignment3;

public class Range {

	public static void main(String[] args) {
		    int start=2, end=45, sum = 0, current;
//		    printf("Enter the start of the range: ");
//		    scanf("%d", &start);
//		    printf("Enter the end of the range: ");
//		    scanf("%d", &end);
		    current = start;
		    while (current <= end) {
		        sum =sum + current;  
		        current++;      
		    }
		    System.out.printf("Sum of numbers from %d to %d is: %d\n", start, end, sum);

	}

}
